playbooks/acs.yml

```
@@ -402,6 +402,17 @@
           }) }}
       when: acs_is_enterprise
 
+    - name: Add extra amps if provided and required properties are set
+      ansible.builtin.set_fact:
+        amps: >-
+          {{ amps | combine({
+            item.name: {
+              'version': item.version
+            }
+          }) }}
+      loop: "{{ acs_play_repository_extra_amp_downloads | default([]) }}"
+      when: acs_play_repository_extra_amp_downloads is defined and item.version is defined and item.name is defined
```

```suggestion
      when: item.version is defined and item.name is defined
```
I think its not needed